test/uac.js

```diff
@@ -1,44 +1,91 @@
+/* global describe, it */
 'use strict';
-//  var uac = require('../lib/utils'),
-//  assert = require('assert');
-
-// describe('UAC elevate tests', () => {
-//     it('Empty file path, should return', () => {
-//         utils.elevate('', null, function (err, result) {
-//             assert.equal(result, true);
-//         });
-//     });
-
-//     it('Missing file path, should return', () => {
-//         utils.elevate(null, null, function (err, result) {
-//             assert.equal(result, true);
-//         });
-//     });
-
-//     it('Should get results for elevate for a given file', (done) => {
-//         utils.elevate('C:\\Program Files\\nodejs\\node.exe', null, function (err, result) {
-//             try {
-//                 assert.ok(true);
-//                 console.log('callback');
-//                 assert(result !== null);
-//                 done();
-//             } catch (x) {
-//                 done(x);
-//             }
-//         });
-//     });
-
-//     it('Should get results for elevate for a given file and parameter', (done) => {
-//         utils.elevate('C:\\Program Files\\nodejs\\node.exe', '--version', function (err, result) {
-//             try {
-//                 assert.ok(true);
-//                 console.log('callback');
-//                 assert(result !== null);
-//                 done();
-//             } catch (x) {
-//                 done(x);
-//             }
-//         });
-//     });
-// });
->>>>>>> fix(lib) - Object-like Key api + Fix Key Value Test
+require('ref');
+var mockery = require('mockery'),
+    assert = require('assert'),
+    struct = require('ref-struct'),
+    uniontype = require('ref-union'),
+    types = require('../lib/types');
+
+var DUMMYUNIONNAME = uniontype({
+    hIcon: types.HANDLE,
+    hMonitor: types.HANDLE
+});
+
+var SHELLEXECUTEINFO  = struct({
+    cbSize: types.DWORD,
+    fMask: types.ULONG,
+    hwnd: types.HWND,
+    lpVerb:  types.STRING,
+    lpFile:  types.STRING,
+    lpParameters: types.STRING,
+    lpDirectory: types.STRING,
+    nShow: types.INT,
+    hInstApp: types.HINSTANCE,
+    lpIDList: types.LPVOID,
+    lpClass: types.STRING,
+    hkeyClass: types.HKEY,
+    dwHotKey: types.DWORD,
+    DUMMYUNIONNAME: DUMMYUNIONNAME,
+    hProcess: types.HANDLE
+});
+
+var shell32Mock = {
+    ShellExecuteExA: function () {
+    }
+};
+shell32Mock.ShellExecuteExA.async = function (type, cb) {
+    console.log('async');
```

Comment:
Let's change the'se console.logs to `debug()` statements using [debug](http://npmjs.org/debug). I'm already doing this in my local changes that will be PR'd soon